### DISCUSSION TITLE
개발과 관련 없는 블로그는 제외할 수 있나요?

### DISCUSSION THREAD

**Discussion by barley-sfn** [awesome-devblog/awesome-devblog#498] [Q&A] [answered]
https://blog.naver.com/kato75

위 블로그가 등록되어 있는 것 같은데
아무리 봐도 개발과 무관한 블로그로 보여서요.
이런 블로그는 제외할 수 있나요?

**Comment by barley-sfn** [comment]
https://kato75.blog.me/

실제로는 위 링크로 들어가 있군요.

**Comment by sarojaba** [comment, marked as answer]
있습니다.

제 기억에 저 블로그는 스크래치 때문에 등록했고, 관련없는 글은 카테고리로 필터링했던것 같습니다. 그 이후로 카테고리가 더 생겼네요.

삭제해도 될것 같네요. 풀리퀘 날려주셔도 됩니다.
